libs/language-server/src/lib/constraint/range-constraint-meta-inf.ts

```diff
@@ -96,5 +96,26 @@ export class RangeConstraintMetaInformation extends ConstraintMetaInformation {
         }
       },
     );
+    super.docs = {
+      description:
+        'Limits the range of a number value with an upper and/or lower boundary. Only values within the given range are valid.',
+      examples: [
+        {
+          description: 'An integer scale between 1 and 100.',
```

Comment:
Since the constraint could possibly be applied to a `decimal` valuetype as well:

```suggestion
          description: 'A scale between 1 and 100.',
```